src/Contracts/SettingContract.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Scout Extended.
+ *
+ * (c) Algolia Team <contact@algolia.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Algolia\ScoutExtended\Contracts;
+
+interface SettingContract
+{
+    public static function exist(string $key, $value, array $array): array;
```

Comment:
Add docs on the interface.

Reply:
not sure i've did it correctly need review @nunomaduro